src/prefect/cli/register.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+import click
+
+import prefect
+from prefect.utilities.storage import extract_flow_from_file
+
+
+@click.group(hidden=True)
+def register():
+    """
+    Register flows
+    """
+
+
+@register.command(
+    hidden=True,
+    context_settings=dict(ignore_unknown_options=True, allow_extra_args=True),
+)
+@click.option(
+    "--file",
+    "-f",
+    required=True,
+    help="A file that contains a flow",
+    hidden=True,
+    default=None,
+    type=click.Path(exists=True),
+)
+@click.option(
+    "--name",
+    "-n",
+    required=False,
+    help="The name of a flow to pull out of the file provided.",
```

Comment:
Is this the `flow.name`, or the python variable name of the flow?

Reply:
`flow.name`, can be documented better